CONTRIBUTING.md

```diff
@@ -1,7 +1,10 @@
 Contributing to Survey Data Collection repositories
 ===================================================
 
-* We use git-flow - create a feature branch from develop, e.g. feature/new-feature
+* We use github-flow - create a branch from master with a short, concise, name, e.g. (nb. there are no specific prefixes):
```

Comment:
I think we should link to http://scottchacon.com/2011/08/31/github-flow.html as it does a good job of explaining some of the reasoning behind github-flow